### ISSUE TITLE
testdrive: rewrite tests to use `PEEK AT <timestamp>` and delete retry loop

### ISSUE THREAD

**Issue by quodlibetor** [MaterializeInc/materialize#1470]
per https://github.com/MaterializeInc/materialize/pull/1469#issuecomment-572218795

**Comment by antiguru** [comment]
This is likely superseded by serializability. Blocked on real-time recency.
